R/pkg/R/DataFrame.R

```diff
@@ -3058,7 +3057,7 @@ setMethod("str",
 #' @note drop since 2.0.0
 setMethod("drop",
           signature(x = "SparkDataFrame"),
-          function(x, col, ...) {
```

Comment:
right, in fact, this one was added in #14705 - which we missed and shouldn't be added.